src/EnergyPlus/ElectricPowerServiceManager.cc

```diff
@@ -2554,32 +2653,32 @@ namespace EnergyPlus {
 		qdotRadZone_       = 0.0;
 	}
 
-	void 
+	void
 	ACtoDCConverter::reinitZoneGainsAtBeginEnvironment()
 	{
 		qdotConvZone_            = 0.0;
 		qdotRadZone_             = 0.0;
 	}
 
-	Real64 
+	Real64
 	ACtoDCConverter::thermLossRate() const
 	{
 		return thermLossRate_;
 	}
 
-	Real64 
+	Real64
 	ACtoDCConverter::dCPowerOut() const
 	{
 		return dCPowerOut_;
 	}
```

Comment:
I know you didn't write these with your PR here.  But my comment about unnecessary getters holds.  More importantly, the casing of these is gross.  dC and aC?  Should be: getDCPowerOut and getACPowerOut, etc.

Reply:
Yeah, that is ugly. I'm wishing my IDE hadn't "cleaned up" the trailing whitespace and you would have never seen it.

Reply:
Ugly, but nothing you added.  If you want to use your IDE to rapidly rename them then that would be great.  But it's not directly in your scope, so your choice.